examples/petsc/bps.c

```diff
@@ -26,6 +26,7 @@
 //     ./bps -problem bp6 -degree 3 -ceed /gpu/cuda
 //
 //TESTARGS -ceed {ceed_resource} -test -problem bp5 -degree 3 -ksp_max_it_clip 15,15
+//TESTARGS -ceed {ceed_resource} -test -problem bp5 -degree 3 -ksp_max_it_clip 50,50 -simplex
```

Comment:
`tap.sh` will need the few lines of multiple-test handling to be moved here. `make junit` handles it correctly already, but `make test search=petsc-bps` will fail.
```suggestion
//TESTARGS(name="bp5 hex") -ceed {ceed_resource} -test -problem bp5 -degree 3 -ksp_max_it_clip 15,15
//TESTARGS(name="bp5 tet") -ceed {ceed_resource} -test -problem bp5 -degree 3 -ksp_max_it_clip 50,50 -simplex
```

Reply:
In a separate PR, I think we should get rid of `tap.sh` (too messy to maintain) and teach `junit.py` to print TAP output (should be fairly easy).

Reply:
How about `multigrid.c`? Do we need to add name? we have simplex there as well.

Reply:
I'm not sure that test names can have spaces in them - the regex isn't super robust.

Reply:
Re `tap.sh`, see: https://gitlab.com/micromorph/ratel/-/merge_requests/226